src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.ts

```diff
@@ -64,6 +64,7 @@ export class LoginComponent implements OnInit {
   }
 
   login() {
+    localStorage.setItem('cluster_api_url', window.location.origin);
```

Comment:
yup, there will be. we should definitely look for alternatives for this one. But since this setting is also being stored in the dashboard settings and retrieved everytime in the navigation component and then setting (or copying) it to localStorage just for the api-intercepting purposes, its probably okay.

But my concern for token is still present

Reply:
@aaSharma14 why are we setting the `cluster_api_url` to this value when we login?